src/doc/unstable-book/src/language-features/rustc-private.md

```diff
@@ -6,6 +6,9 @@ The tracking issue for this feature is: [#27812]
 
 ------------------------
 
-This feature allows access to unstable internal compiler crates.
+This feature allows access to unstable internal compiler crates such as `rustc_driver`.
 
-Additionally it changes the linking behavior of crates which have this feature enabled. It will prevent linking to a dylib if there's a static variant of it already statically linked into another dylib dependency. This is required to successfully link to `rustc_driver`.
+The presence of this feature changes the way the linkage format for dylibs is calculated in a way
+that is necessary for linking against dylibs that statically link `std` (such as `rustc_driver`).
+This makes this feature "viral"; its use in a particular crate makes its use required in all
+dependent crates (including integration tests, which are built as separate crates).
```

Comment:
Technically it is only necessary in crates that actually get linked.

Reply:
How would you phrase that?

Reply:
Updated, PTAL.